cli/tests/unit/os_test.ts

```diff
@@ -185,3 +185,19 @@ unitTest({ perms: { env: false } }, function releasePerm(): void {
   }
   assert(caughtError);
 });
+
+unitTest({ perms: { env: true } }, function freemem(): void {
+  assertNotEquals(Deno.freemem(), "");
+});
+
+unitTest({ perms: { env: false } }, function freememPerm(): void {
```

Comment:
I'm not sure about the env permission...

Reply:
Mee to. I just opted for the same permission used by other `os` functions.